src/test/utils.ts

```diff
@@ -149,6 +150,60 @@ export function runTestWithoutSMAccess() {
   });
 }
 
+export function runTestAsRbacReader() {
```

Comment:
Nit: can we uppercase all the RBAC in the function name (`runTestAsRBACReader`)? (and for the functions below).